thread-random/src/Counter.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Random;
+
+public class Counter implements Runnable {
+    private static int count = 0;
+    private int id;
+
+    public Counter(int id)
+    {
+        this.id = id;
+    }
+
+    @Override
+    public void run()
+    {
+        try {
+            int ms = new Random().nextInt(6) + 5;
+            Thread.sleep(ms * 1000);
+            synchronized (this) {
```

Comment:
@tatevikb Move increment block to independent method

Reply:
Done